rubicon-htb.js

```diff
@@ -765,7 +797,8 @@ function RubiconModule(configs) {
         setFirstPartyData: setFirstPartyData,
 
         //? if (TEST) {
-        __parseResponse: __parseResponse
+        parseResponse: __parseResponse,
+        generateRequestObj: __generateRequestObj
```

Comment:
Please change this back.

Reply:
Yes but the test file is using: partnerModule.generateRequestObj and partnerModule. parseResponse
Will update then to:
partnerModule.__generateRequestObj
partnerModule.__ parseResponse

Reply:
Same comment as above.